packages/python3.rb

```diff
@@ -70,12 +72,12 @@ def self.build
     # test_urllib, test_urllib2, test_urllib2_localnet.
     # So, modifying environment variable to make pass tests.
 
-    Dir.mkdir 'builddir'
+    FileUtils.mkdir_p 'builddir'
     Dir.chdir 'builddir' do
       system CREW_ENV_OPTIONS_HASH.transform_values { |v| "#{v} #{@cppflags}" }, "../configure #{CREW_OPTIONS} \
         --with-computed-gotos \
         --enable-loadable-sqlite-extensions \
-        --without-ensurepip \
+        --with-ensurepip \
```

Comment:
I agree.  We should be bundling pip with python.  That said, I think python needs to be removed prior to building so it integrates correctly.  Does this mean the `py3_pip` package goes away?

Reply:
That would be the idea. It would be nice to just make crew a frontend for pip3 for python3 packages...